errata_tool/release.py

```diff
@@ -164,3 +174,11 @@ def create(klass, name, product, product_versions, type, program_manager,
             print(body)
             raise ReleaseCreationError('see field_errors <div>')
         return klass(name=name)
+
+
+def unquote_plus(string):
```

Comment:
should probably put together some basic unittest coverage of this limited parsing function

Reply:
I'll rename this method to start with an underscore so it's clearer that we mean this method to be private, and I'll add a comment that we'll delete it eventually.

The final commit in this PR "release: support url-encoded "+" chars in names" adds the unit test that exercises this. I'll rebase this on the latest master (picking up #211) so it's easier to see what the test coverage is.